servers/zts/src/main/java/com/yahoo/athenz/zts/cert/impl/DynamoDBCertRecordStoreConnection.java

```diff
@@ -139,10 +156,10 @@ public boolean updateX509CertRecord(X509CertRecord certRecord) {
                             new AttributeUpdate(KEY_CURRENT_SERIAL).put(certRecord.getCurrentSerial()),
                             new AttributeUpdate(KEY_CURRENT_IP).put(certRecord.getCurrentIP()),
                             new AttributeUpdate(KEY_CURRENT_TIME).put(getLongFromDate(certRecord.getCurrentTime())),
+                            new AttributeUpdate(KEY_CURRENT_DATE).put(dynamoDBUtils.getIso8601FromDate(certRecord.getCurrentTime())),
```

Comment:
Everytime we update the currentTime column (hold epoch in long format), we should also update the new currentDate column (ISO date in the format YYYY-MM-DD).